canonical-web-and-design/charmhub.io
# Upgrade vanilla to 2.9.0 & add side navigation to home page

## Done

- Upgrade to vanilla 2.9.0
- Add vanilla side-navigation to the homepage

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the side navigation works and looks as expected


## Issue / Card

Fixes #31 


## Note

@spencerbygraves  as there are no such colours for the side navigation in vanilla, do we want to stick to the default colours, or we want to implement charmhub colours in vanilla?
